### DISCUSSION TITLE
HTML snippets in documents

### DISCUSSION THREAD

**Discussion by ghost** [nvim-neorg/neorg#676] [Q&A] [answered]
I couldn't find anything about embedding literal HTML in documents.

I believe in Org it is possible via `#+HTML` and markdown supports HTML as well.
Is there any way to do this in norg?

**Comment by vhyrro** [comment, marked as answer]
This can be done with the `@embed` tag :)
```
@embed html
<p>Here it is</p>
@end
```

**Reply by ghost** [reply]
Wow that was fast :) awesome, thanks!
